trade-finance-manager-ui/server/controllers/utilisation-reports/helpers/edit-payment-helper.test.ts

```diff
@@ -367,6 +379,31 @@ describe('edit-payment-helper', () => {
       // Assert
       expect(viewModel.formValues.paymentReference).toBe(reference);
     });
+
+    it('sets the view model redirectTab to the supplied redirectTab', () => {
+      // Arrange
+      const redirectTab = RECONCILIATION_FOR_REPORT_TABS.KEYING_SHEET;
+
+      // Act
+      const viewModel = getEditPaymentViewModel(aPaymentDetailsWithFeeRecordsResponseBody(), reportId, paymentId, isCheckboxChecked, undefined, redirectTab);
+
+      // Assert
+      expect(viewModel.redirectTab).toEqual(redirectTab);
+    });
+
+    it('sets the back link to the reconciliation for report href using the supplied redirectTab', () => {
+      // Arrange
+      const redirectTab = RECONCILIATION_FOR_REPORT_TABS.KEYING_SHEET;
+
+      jest.mocked(getReconciliationForReportHref).mockReturnValue('/utilisation-reports/123#keying-sheet');
+
+      // Act
+      const viewModel = getEditPaymentViewModel(aPaymentDetailsWithFeeRecordsResponseBody(), reportId, paymentId, isCheckboxChecked, undefined, redirectTab);
```

Comment:
Not a blocker as this is an established pattern with the view model functions - these could have object structured params - then avoids having to pass in `undefined` params

Can improve these in the future